PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java

```diff
@@ -270,6 +188,88 @@ public void onClick(View v) {
         getmDbHandler().close();
     }
 
+    @Override
+    protected void onSaveInstanceState(Bundle savedInstanceState) {
+        super.onSaveInstanceState(savedInstanceState);
+        savedInstanceState.putInt("eyeNum", eye);
+        savedInstanceState.putInt("faceNum", face);
+        savedInstanceState.putInt("hairNum", hair);
+        savedInstanceState.putInt("clothNum", cloth);
+    }
+
+    @Override
+    protected void onRestoreInstanceState(Bundle savedInstanceState) {
+        super.onRestoreInstanceState(savedInstanceState);
```

Comment:
These values are generated when the user clicks the left and right chooser buttons. Please correct me if I misunderstood something.